calico-vpp-agent/cmd/calico_vpp_dataplane.go

```diff
@@ -93,10 +96,47 @@ func main() {
 	}
 	// Once we have the api connection, we know vpp & vpp-manager are running and the
 	// state is accurately reported. Wait for vpp-manager to finish the config.
-	err = common.WaitForVppManager()
+	vppManagerInfo, err := common.WaitForVppManager()
 	if err != nil {
 		log.Fatalf("Vpp Manager not started: %v", err)
 	}
+	routeWatcher := &watchers.RouteWatcher{}
+	poolWatcher := &watchers.PoolWatcher{
+		RouteWatcher:     routeWatcher,
+		UserSpecifiedMtu: vppManagerInfo.UserSpecifiedMtu,
+		Mtu:              vppManagerInfo.MainUplinkMtu,
+		FakeNextHopIP4:   vppManagerInfo.FakeNextHopIP4,
+		FakeNextHopIP6:   vppManagerInfo.FakeNextHopIP6,
+	}
+	go routeWatcher.WatchRoutes()
```

Comment:
We should probably stick to the `Go(routeWatcher.WatchRoutes)` syntax and have the tomb as a param to handle graceful teardown